tests/foreman/longrun/test_inc_updates.py

```diff
@@ -158,17 +159,17 @@ def host(
     )
     # Add filter of type include but do not include anything.
     # this will hide all RPMs from selected erratum before publishing.
-    entities.RPMContentViewFilter(
+    module_target_sat.api.RPMContentViewFilter(
         content_view=module_cv, inclusion=True, name='Include Nothing'
     ).create()
     module_cv.publish()
     module_cv = module_cv.read()
     return rhel7_contenthost_module
 
 
-def get_applicable_errata(repo):
+def get_applicable_errata(satellite, repo):
```

Comment:
We could probably get rid of this helper completely since it's just an one-liner and used just once.
```suggestion
```
I understand this is out of scope of this PR, non-blocking.

Reply:
Thank you @tpapaioa !